src/Package/Impl/DataInspect/VariableGridWindowPane.cs

```diff
@@ -24,5 +24,10 @@ internal sealed class VariableGridWindowPane : RToolWindowPane {
             }
             _gridHost.SetEvaluation(evaluation);
         }
+
+        protected override void OnClose() {
+            base.OnClose();
+            _gridHost.CleanUp();
```

Comment:
Also verify detaching the tab.

---
In reply to: [67448357](https://github.com/Microsoft/RTVS/pull/1963#discussion_r67448357) [](ancestors = 67448357)